.github/workflows/e2e_test.yml

```diff
@@ -12,7 +12,10 @@ permissions:
   packages: write
 
 jobs:
-  test-ibc-transfer:
+  e2e-test:
+    strategy:
+      matrix:
+        tests: [e2e-test-ibc, e2e-test-ibc-timeout]
```

Comment:
When our test base grows, do we need to specify every test here?

Reply:
I think so, we will need to add our test case names here as we develop them. I might expand it to automatically collect these commands